iroha-lib/model/TxBatch.cpp

```diff
@@ -0,0 +1,42 @@
+#include "TxBatch.hpp"
+
+#include "model/converters/pb_common.hpp"
+
+
+namespace iroha_lib
+{
+
+iroha::protocol::Transaction::Payload::BatchMeta::BatchType TxBatch::getBatchType(bool atomic) const
```

Comment:
This function, if not marked `inline` is going to cause trouble. It should be implemented either in the header file (as is) or marked as such.

Reply:
Done.